app/src/main/res/values/strings.xml

```diff
@@ -7,4 +7,126 @@
     <string name="title_profile">Profile</string>
     <string name="title_activity_maps">Map</string>
     <string name="imageHolder">imageHolder</string>
+    <string name="terms_and_conditions">By downloading or using the app, these terms will automatically apply to you – you should make
```

Comment:
Is there another way to store this huge string? 
If not please create a task for this.

Reply:
I think it's possible to load from the database or from a txt file? Which one do you think is better?

Reply:
The screenshots from IOS also indicate that the text is shorter(I copied pasted from the documents on Google drive). I wonder if there is a shorter version I should use instead ?

Reply:
Yes the ios Version is shorter.

Reply:
I have changed the string so that it matches the IOS version now. I noticed that the text are switched from that of google drive(the text corresponding to privacy is put in terms and conditions screen and vice-versa). In addition, the email provided in the IOS version is wrong. Should be quick fix either way.